docs/src/modules/components/ApiPage.tsx

```diff
@@ -30,7 +30,7 @@ import {
   getClassApiDefinitions,
   getClassesToC,
 } from 'docs/src/modules/components/ApiPage/definitions/classes';
-import { getSlotsApiDefinitions } from './ApiPage/definitions/slots';
+import { getSlotsApiDefinitions } from 'docs/src/modules/components/ApiPage/definitions/slots';
```

Comment:
That's a post traumatic symptomes.

Did not took care of it so I've little context, but most of the files on `docs/modules` don't have a relative import because it was causing conflict when used in mui-x repo